docs/typescript/default.md

```diff
@@ -0,0 +1,72 @@
+---
+order: 120
+icon: /static/typescript.svg
+label: Typescript
+expanded: true
+---
+
+# Typescript
+
+!!!warning Before you continue...
+
+The prefered way for using these [TypeScript](https://www.typescriptlang.org/) shared configurations is **not** by installing them manually, but rather by **scaffolding** your application with Workleap's [foundry-cli](https://github.com/gsoft-inc/wl-foundry-cli).
+
++++ pnpm
+```bash
+pnpm create @workleap/project@latest <output-directory>
+```
++++ yarn
+```bash
+yarn create @workleap/project@latest <output-directory>
+```
++++ npm
+```bash
+npm create @workleap/project@latest <output-directory>
+```
++++
+!!!
+
+## By project type configurations
+
+Typically, [TypeScript](https://www.typescriptlang.org/) shareable configuration libraries are architectured around the idea that shared configurations should be small and composable to accomodate any potential use cases that a development team might encounter.
+
+While composability provide flexibility, it also increase the complexity on the consumer side as they must understand how [TypeScript configuration inheritance](https://www.typescriptlang.org/tsconfig#extends) works and how to compose the provided shared configurations correctly. **It can be frustrating at times** when you want to get something up and running quickly.
```

Comment:
```suggestion
While composability provides flexibility, it also increases the complexity on the consumer side as they must understand how [TypeScript configuration inheritance](https://www.typescriptlang.org/tsconfig#extends) works and how to compose the provided shared configurations correctly. **It can be frustrating at times** when you want to get something up and running quickly.
```